babel/dates.py

```diff
@@ -1272,8 +1276,11 @@ def format_weekday(self, char, num):
                 return self.format(value % 7 + 1, num)
             num = 3
         weekday = self.value.weekday()
-        width = {3: 'abbreviated', 4: 'wide', 5: 'narrow'}[num]
-        context = {3: 'format', 4: 'format', 5: 'stand-alone'}[num]
+        width = {3: 'abbreviated', 4: 'wide', 5: 'narrow', 6: 'short'}[num]
+        if char == 'c':
```

Comment:
Can you add some sort of reference about the semantics of lower-case `c` using stand-alone context? It doesn't seem immediately obvious, esp. considering this is a format function...